.travis.yml

```diff
@@ -35,5 +35,5 @@ script:
     - flake8 tests
     - flake8 examples
     - python tests/test_peninsula.py --grid 20 10
-    - py.test -v -s tests/
+    - py.test tests/
```

Comment:
Hehe, I was getting annoyed with reading so many compilation messages, but sure we can drop this change until our test base has grown a bit further.